Wikipedia/assets/codemirror/resources/addon/search/search.js

```diff
@@ -82,7 +95,7 @@
       state.queryText = query;
       state.query = parseQuery(query);
       cm.removeOverlay(state.overlay, queryCaseInsensitive(state.query));
-      state.overlay = searchOverlay(state.query, queryCaseInsensitive(state.query));
+      state.overlay = searchOverlay(cm, state, state.query, queryCaseInsensitive(state.query));
```

Comment:
now that `searchOverlay` is taking the entire `state`, it could get `query` itself instead of taking it as a separate parameter

Reply:
Yep good catch, this is done too.